components/GroupCard.tsx

```diff
@@ -0,0 +1,42 @@
+import Tag from '../components/Tag';
+import Link from 'next/link';
+
+type Props = {
+  title: string;
+  subtitle?: string;
+  description: string;
+  genres: Array<string>;
+  /**path to group page */
+  href: string;
+};
+
+export default function GroupCard({
+  title,
+  subtitle,
+  description,
+  genres,
+  href,
+}: Props) {
+  return (
+    <div className="border rounded-xl px-6 py-5">
+      <div className="flex items-center gap-3 border-b pb-4">
+        <div className="w-10 h-10 rounded-full bg-black"></div>
+        <div className="flex flex-col">
+          <span>{title}</span>
+          <span className="text-sm text-gray-500">{subtitle}</span>
```

Comment:
I think th DB will return the number of members as a number like this : 

```ts
{
nbMembers : 4
}
```

so maybe the props should be : 

```ts
type Props = {
  title: string;
  nbMembers: number;
  description: string;
  genres: Array<string>;
  /**path to group page */
  href: string;
};
```

and the div like this : 

```tsx
<span className="text-sm text-gray-500">{nbMembers} membre{nbMembers>1 ? 's' : ''}</span>
```

What do @alexandrelam  think ?